controllers/v1/reportsController.js

```diff
@@ -2598,4 +2575,51 @@ module.exports = class Reports {
     })
     return schoolFieldArray;
   }
+
+  evaluationFrameworkDocument(evaluationFrameworksDocuments) {
```

Comment:
Please move this to util file for common use. Also, try to achieve this in one function. @aman-tunerlabs

Reply:
Added functionality in utils. EvaluationFrameworkDocuments will be refering to generatePathToCriteria in utils.